data/gallery.js

```diff
@@ -7,22 +7,40 @@ const album = {
         {
             title: `Photo 1`,
             src: `./gallery/IMG_0945.jpg`,
-            created: `4th Oct 2017`,
+            created: new Date(1507109400 * 1000),
             description: `Waliking down the aisle `,
```

Comment:
worth changing this to

```
created: 1507109400
```

then later on, when you need to show the date:

```
new Date(album.images[0].created * 1000);
```

although what you've got here will work, you can't store date objects in databases or in json strings.